plugins/sfc/renderer/l2xconn/l2xconn_renderer.go

```diff
@@ -139,10 +139,11 @@ func (rndr *Renderer) renderChain(sfc *renderer.ContivSFC, isDelete bool) (confi
 
 		if iface != "" && prevIface != "" {
 			if rndr.isNodeLocalSF(sf) && rndr.isNodeLocalSF(prevSF) {
-				// the two SFs (prevSF and SF) are local - cross-connect the interfaces
-				xconnect := rndr.crossConnectIfaces(prevIface, iface, sfc.Unidirectional)
-				rndr.mergeConfiguration(config, xconnect)
-
```

Comment:
Can be simplified to one-level `if`:
```
if rndr.shouldChainLocalSFs(sf, prevSF) {
    xconnect := rndr.crossConnectIfaces(prevIface, iface, sfc.Unidirectional)
    rndr.mergeConfiguration(config, xconnect)
}  else if rndr.shouldChainToRemoteSF(prevSF, sf) {
...
```

Also comment `... - Rgarding the node ID (lower or note) cross-connect the interfaces` doesn't make much sense.

Reply:
Yes its OK, I have committed again the right code